sonarqube-sample-java/src/main/java/com/consoleconnect/mw/sonarqube/sample/entity/Student.java

```diff
@@ -0,0 +1,20 @@
+/**
+ * 
+ */
+package com.consoleconnect.mw.sonarqube.sample.entity;
+
+/**
+ * @author dxiong
+ *
+ */
+public class Student {
+
+  
+  public String name() {
+    boolean flag = false;
+    while(flag) {
+      System.out.println("hello");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sq.ci.consoleconnect.com/coding_rules#rule_key=squid%3AS106)